<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1351</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>210</y>
      <width>501</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 250px 30px;
float: left;
border: 5px solid rgb(85, 255, 0);
background-color: rgb(13, 39, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>501</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 250px 30px;
float: left;
border: 5px solid rgb(85, 255, 0);
background-color: rgb(41, 115, 0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="txtNamaPetugas">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>421</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script MT Bold</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
border: 3px solid transparent;
border-bottom-color:  gold;
color: gold;
padding-bottom: 7px;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Nama Pengguna</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPassword">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>421</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script MT Bold</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
border: 3px solid transparent;
border-bottom-color:  gold;
color: gold;
padding-bottom: 7px;</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Password Pengguna</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSubmit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>501</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 250px 20px;
float: left;
border: 5px solid rgb(85, 255, 0);
background-color: rgb(13, 39, 0);
color: gold;</string>
      </property>
      <property name="text">
       <string>L o g i n</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>70</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(&quot;icons/user.png&quot;) 0 0 0 0 stetch stretch;
border-radius: 50px;
border: transparent;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>200</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(&quot;icons/pwd.png&quot;) 0 0 0 0 stetch stretch;
border-radius: 50px;
border: transparent;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script MT Bold</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid transparent;
background: transparent;
color: rgb(85, 255, 0);</string>
      </property>
      <property name="text">
       <string>Masukan Nama Pengguna</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script MT Bold</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid transparent;
background: transparent;
color: rgb(85, 255, 0);</string>
      </property>
      <property name="text">
       <string>Masukan Password Pengguna</string>
      </property>
     </widget>
     <zorder>txtPassword</zorder>
     <zorder>txtNamaPetugas</zorder>
     <zorder>btnSubmit</zorder>
     <zorder>frame_3</zorder>
     <zorder>frame_4</zorder>
     <zorder>label_13</zorder>
     <zorder>label_14</zorder>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Script MT Bold</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid transparent;
background: transparent;
color: gold;</string>
     </property>
     <property name="text">
      <string>Silakan Login</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
